### ISSUE TITLE
Support for 5.4?

### ISSUE THREAD

**Issue by seltzdesign** [microdee/OpenUnrealSpaceMouse#109]
Hello David.

Are you going to release a version of the plugin for UE5.4? What would it take?

Thanks.

**Comment by microdee** [comment]
Unfortunately these days I don't really have time for this plugin as my
current work/interests are not involving daily 3D navigation at all. I
think I will make it free on Marketplace and will show a disclaimer in the
readme.

**Comment by seltzdesign** [comment]
Fair enough. It's the one plugin I always enable at the start of a new UE project, because I am so used to the spacemouse. I guess I can create a fork and try to build it for UE5.4. If we manage to do it, I can make a PR - hopefully it's pretty straight forward.

**Comment by seltzdesign** [comment]
I guess there is 3Dconnexion's own plugin, but I'd have to install their crappy 3DxWare, which will interfere with other tools. Maybe I can create a separate Windows user where the driver is installed for UE5.4 only.

**Comment by ani-picsel100** [comment]
> Fair enough. It's the one plugin I always enable at the start of a new UE project, because I am so used to the spacemouse. I guess I can create a fork and try to build it for UE5.4. If we manage to do it, I can make a PR - hopefully it's pretty straight forward.

If you succeed in building version 5.4, can you upload it? It's a plugin I've been using, but I don't have the ability to build it and use it. Thank you.

**Comment by microdee** [comment]
Good news, there's a project I'm working on where space-mice are also needed, so expect a big 2.0 update on this. At least the code will be a lot more elegant

**Comment by microdee** [comment]
yeah when I get there it'll target newer engine versions

**Comment by microdee** [comment]
Fyi this project is now back in business! With fresh [1.3](https://github.com/microdee/OpenUnrealSpaceMouse/releases/tag/1.3.1) and a [2.0](https://github.com/microdee/OpenUnrealSpaceMouse/releases/tag/2.0.1) releases support up until UE 5.7
